src/GUI/EFCorePowerTools/ViewModels/ColumnInformationViewModel.cs

```diff
@@ -74,6 +75,18 @@ public bool IsPrimaryKey
             }
         }
 
+        public bool IsForeignKey
+        {
+            get => _isForeignKey;
+            set
+            {
+                if (Equals(value, _isForeignKey)) return;
+                _isForeignKey = value;
+                RaisePropertyChanged();
+                RaisePropertyChanged(nameof(IsEnabled));
```

Comment:
nameof (IsForeignKey) ? and why two x Raise ??

Reply:
It's correct like it is. 
The first call to RPC automatically infers the name from the property name so uses "IsForeignKey". The second call to RPC forces to reevaluate the IsEnabled property since it depends on IsForeignKey.